src/audio/audio_sound.js

```diff
@@ -3,6 +3,26 @@ pc.extend(pc, function () {
 
     var Sound;
 
+    // checks if user is running IE
+    var ie = function () {

```

Comment:
You can pre-cache result of this function, as userAgent does not changes in real-time.
So you wont run it on every audio element added. As doing indexOf of relatively long userAgent, can be costly for real-time.